cwms-servers/CWMS_HOME/bin/cwmsClasspath

```diff
@@ -0,0 +1,61 @@
+#!/bin/env python
+'''
+Script:	cwmsClasspath
+
+Usage:	cwmsClasspath [/check] [classpathItem] [classpathItem] ...
+
+Where:	classpathItem = {"classes"|{{"jars"}[:specificItem][,specificItem]...}}
+	specificItem = [-]jarFileName
+
+If no classpathItem is specified, the classpath items default to "classes and jars"
+
+The resulting classpath is written to stdout, so it can be included in shell scripts.
+
+Notes and warnings are written to stderr.
+
+The contents of the jar directory is checked against lists of expected
+jar files.  Any discrepancies are output as warnings.  Jars file that are not listed
+in the expected list are included in the resulting classpath.
+
+The /check option checks every class in the resulting classpath to see if it exists
+in more than one location and outputs warnings about classes defined in multiple locations.
+This is useful for interactively checking the contents of the classes and jars
+directories by executing "cwmsClasspath /check"
+
+Mike Perryman
```

Comment:
Do you want to keep this header comments as is including referencing Mike P

Reply:
Updated